BootloaderCorePkg/Tools/GenContainer.py

```diff
@@ -234,7 +235,10 @@ def calculate_auth_data (file, auth_type, priv_key, out_dir, hash_type = 'SHA2_2
         elif auth_type in ["SHA2_256"]:
             data = get_file_data (file)
             hash_data.extend (hashlib.sha256(data).digest())
-        elif auth_type in ['RSA2048']:
+        elif auth_type in ["SHA2_384"]:
+            data = get_file_data (file)
+            hash_data.extend (hashlib.sha384(data).digest())
+        elif auth_type in ['RSA2048', 'RSA3072']:

```

Comment:
Do we need to verify if the key is actually matching the auth_type ?   EX: when auth_type is RSA3072, the actual key file should be also RSA3072 key.